features/background-sync.yml

```
@@ -0,0 +1,13 @@
+name: Background sync
+description: The `ServiceWorkerRegistration.sync` API registers a service worker callback to run only when the device is no longer offline. You can use this to let users continue using your app while offline, and synchronize with a server later when, when a connection is established.
```

```suggestion
description: The background synchronization API registers a service worker callback to run only when the device is no longer offline. You can use this to let users continue using your app while offline, and synchronize with a server later when, when a connection is established.
```

See https://github.com/web-platform-dx/web-features/pull/1363#discussion_r1675652337. @captainbrosset can you review? I don't want to have a suggestion battle :D

---

I agree with your comment. The word SericeWorkerRegistration won't appear in code, and sync is too short. Let's stick with background synchronization API. More recognizable.